classes/Http/Controller/ForgotController.php

```diff
@@ -40,20 +55,14 @@ public function indexAction()
 
     public function sendResetAction(Request $request)
     {
-        /** @var FormFactory $formFactory */
-        $formFactory = $this->service('form.factory');
-
-        $form = $formFactory
+        $form = $this->formFactory
             ->createBuilder(ForgotForm::class)
             ->getForm();
 
         $form->handleRequest($request);
 
-        /** @var Session\Session $session */
-        $session = $this->service('session');
```

Comment:
Since we would otherwise be invoking `$request->getSession()` multiple times, can we maybe just use `$session` here and assign it `$request->getSession()`?

Reply:
See the discussion above.

Reply:
See the discussion above.